attributes/default.rb

```diff
@@ -48,6 +48,7 @@
 apache2['additional_config'] = []
 apache2['logs']['access'] = apache2['logs_dir'] + 'cachethq.access.log'
 apache2['logs']['error'] = apache2['logs_dir'] + 'cachethq.error.log'
+apache2['default_conf'] = '/etc/apache2/sites-available/000-default.conf'
```

Comment:
please append to attribute `apache2['confdir']`

Reply:
done